### PR TITLE
AV-183113: Updated k8s version to 1.27 in buildsettings

### PR DESCRIPTION
**Pull request by saurabhvaidvmware**

### CODE REVIEW

**Comment by saurabhvaidvmware** [comment]
build ako

**Comment by swathinsankaran** [comment]
@saurabhvaidvmware could you please update the version to 1.11.1 in `version.yaml`?

File: buildsettings.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "minVersion": "21.1.5"
     },
     "kubernetes": {
-        "maxVersion": "1.26",
+        "maxVersion": "1.27",
         "minVersion": "1.22"
```
Comment (swathinsankaran) [inline buildsettings.json:9]: @akshayhavile should we update the `minVersion` based on the TKG?

**Comment by saurabhvaidvmware** [comment]
> @saurabhvaidvmware could you please update the version to 1.11.1 in `version.yaml`?

Should I change the version in buildsettings too or leave it as is for now? @swathinsankaran

**Comment by swathinsankaran** [comment]
@saurabhvaidvmware Yes, please update the buildsettings.json also. You can leave the version in the documents as it is.

**Comment by saurabhvaidvmware** [comment]
build ako

**Review by swathinsankaran — APPROVED** [review]
LGTM.

MinVersion can be changed later if required.

**Pull request merged**